src/libZilliqa/Zilliqa.cpp

```diff
@@ -281,6 +281,8 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, bool loadConfig,
       case SyncType::DB_VERIF:
         LOG_GENERAL(INFO, "Intitialize DB verification");
         m_n.ValidateDB();
+        std::this_thread::sleep_for(std::chrono::seconds(10));
+        raise(SIGKILL);
```

Comment:
Doesn't `ValidateDB` already call `raise(SIGKILL)`?

Reply:
no I removed it from there as the function might return false and then it is not invoked